src/main/java/net/dries007/tfc/util/OreDictionaryHelper.java

```diff
@@ -126,6 +129,54 @@ public static void init()
+                    OreDictionary.registerOre("craftingToolMediumAxe", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolMediumPick", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+
+                }
+                if (metal.getTier().isAtLeast(Metal.Tier.TIER_V))
+                {
+                    OreDictionary.registerOre("craftingToolHammer", new ItemStack(ItemMetal.get(metal, Metal.ItemType.HAMMER), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolMediumHammer", new ItemStack(ItemMetal.get(metal, Metal.ItemType.HAMMER), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolHardHammer", new ItemStack(ItemMetal.get(metal, Metal.ItemType.HAMMER), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolChisel", new ItemStack(ItemMetal.get(metal, Metal.ItemType.CHISEL), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolMediumChisel", new ItemStack(ItemMetal.get(metal, Metal.ItemType.CHISEL), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolHardChisel", new ItemStack(ItemMetal.get(metal, Metal.ItemType.CHISEL), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolSaw", new ItemStack(ItemMetal.get(metal, Metal.ItemType.SAW), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolMediumSaw", new ItemStack(ItemMetal.get(metal, Metal.ItemType.SAW), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolHardSaw", new ItemStack(ItemMetal.get(metal, Metal.ItemType.SAW), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolAxe", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolPick", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolMediumAxe", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolMediumPick", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolHardAxe", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+                    OreDictionary.registerOre("craftingToolHardPick", new ItemStack(ItemMetal.get(metal, Metal.ItemType.AXE), 1, OreDictionary.WILDCARD_VALUE));
+                }
+            }
+        }
```

Comment:
Those should be on each tool's registration instead of here: https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/0efbf98ad4b35547311ecb5e7d478ecede7f45fa/src/main/java/net/dries007/tfc/objects/items/metal/ItemMetalTool.java#L107-L115

You can simply add this (change for each tool):
```
OreDictionaryHelper.register(this, "crafting", "tool", "Hammer");
if (metal.getTier().isAtLeast(Metal.Tier.TIER_IV))
{
    OreDictionaryHelper.register(this, "crafting", "tool", "medium", "hammer");
}
```